CompatBot/Commands/Misc.cs

```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
 
         private static readonly List<string> EightBallAnswers = new List<string>
         {
-            // 37
+            // 43
             "Ya fo sho", "Fo shizzle mah nizzle", "Yuuuup", "Da", "Affirmative", // 5
             "Sure", "Yeah, why not", "Most likely", "Sim", "Oui",
             "Heck yeah!", "Roger that", "Aye!", "Yes without a doubt m8!", "<:cell_ok_hand:324618647857397760>",
             "Don't be an idiot. YES.", "Mhm!", "Many Yes", "Yiss", "Sir, yes, Sir!", 
             "Yah!", "Ja", "Umu!", "Make it so", "Sure thing",
             "Certainly", "Of course", "Definitely", "Indeed", "Much yes",
             "Consider it done", "Totally", "You bet", "Yup", "Yep",
-            "Certainly", "Yarp",
+            "Positive!", "Yarp", "Hmmm, yes!", "That's a yes for me", "Aye mate",
+            "Totally", "Totes my goats", "Without fail",
 
-            // 20
+            // 24
             "Maybe", "I don't know", "I don't care", "Who cares", "Maybe yes, maybe not",
             "Maybe not, maybe yes", "Ugh", "Probably", "Ask again later", "Error 404: answer not found",
             "Don't ask me that again", "You should think twice before asking", "You what now?", "Ask Neko", "Ask Ani",
             "Bloody hell, answering that ain't so easy", "I'm pretty sure that's illegal!", "What do *you* think?", "Only on Wednesdays", "Look in the mirror, you know the answer already",
+            "Don't know, don't care", "*visible confusion*", "Have you googled it?", "Not sure my dude", 
 
-            // 29
+            // 34
             "Nah mate", "Nope", "Njet", "Of course not", "Seriously no",
             "Noooooooooo", "Most likely not", "Não", "Non", "Hell no",
             "Absolutely not", "Nuh-uh!", "Nyet!", "Negatory!", "Heck no",
             "Nein!", "I think not", "I'm afraid not", "Nay", "Yesn't",
             "No way", "Certainly not", "I must say no", "Nah", "Negative",
-            "Definitely not", "No way, Jose", "Not today", "Narp",
+            "Definitely not", "No way, Jose", "Not today", "Narp", "Not in a million years", 
+            "I’m afraid I can’t let you do that Dave.", "This mission is too important for me to allow you to jeopardize it.", "Oh, I don't think so", "By *no* means",
```

Comment:
you should use regular tick mark for consistency, not every font in every language has fancy curled ones

Reply:
Fixed it